### PR TITLE
Media Unit Testing 2

### PR DESCRIPTION
**Pull request by ianwremmel**
Closes #467

### CODE REVIEW

File: packages/plugin-phone/test/unit/spec/web-rtc-media.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        `audio`,
+        `video`
+      ]
+        .forEach((mediaType) => {
+          [
+            [true, true, false],
+            [true, false, false],
+            [false, true, true],
+            [false, false, true]
+          ].forEach(([sending, receiving, shouldRenegotiate]) => {
```
Comment (adamweeks) [inline packages/plugin-phone/test/unit/spec/web-rtc-media.js]: Wow, that is efficient! Covers lots of cases with very little code.

File: packages/plugin-phone/src/web-rtc-media.js [hunk 2]
```diff
@@ -12,13 +13,64 @@ import {
...
+ * @returns {bool} true if receiving, false if not
+ */
+function getRemoteMediaStatus(kind, pc) {
+  const streams = pc.getRemoteStreams();
+
+  if (streams.length === 0) {
+    return false;
+  }
+
+  const res = streams.reduce((areStreamsFlowing, stream) => {
```
Comment (adamweeks) [inline packages/plugin-phone/src/web-rtc-media.js]: Would a `.some` work better here? I don't think I've used a `.reduce` for a boolean before.

Comment (ianwremmel) [inline packages/plugin-phone/src/web-rtc-media.js]: I hadn't used `some` before. I like it in principle, but my first attempt to switch broke all the tests, so i'm inclined to leave this as-is for now :)

File: packages/plugin-phone/src/webrtc.js [hunk 3]
```diff
@@ -49,87 +30,54 @@ const startSendingMedia = curry((kind, pc) => {
...
-    stream.getTracks().forEach((track) => {
-      if (track.kind === kind) {
-        track.enabled = false;
-        // Store the tracks so we can add them back later (should the user want
-        // to mute/unmute)
-        storeMediaTrackByKindByStream(kind, stream, track);
-        if (pc.removeTrack) {
-          const sender = find(pc.getSenders(), (s) => s.track === track);
-          pc.removeTrack(sender);
+          // The next few lines are a silly hack to deal with chrome not
```
Comment (adamweeks) [inline packages/plugin-phone/src/webrtc.js]: Is there an issue we can reference to check in the future?

Comment (ianwremmel) [inline packages/plugin-phone/src/webrtc.js]: no, but i just wrote myself a note to create one

**Comment by ianwremmel** [comment]
closing in favor of alternate merge method

**Pull request closed without merging**